### ISSUE TITLE
Will not work over Bluetooth Speaker on Pi3

### ISSUE THREAD

**Issue by s10dude** [amzn/alexa-avs-raspberry-pi#94]
Hello, I'm trying to use a wireless Bluetooth speaker with the pi echo. No mater what I do the sound will only come through the HDMI port even though the BT speaker is paired and all Youtube video sounds are coming through it.  Has anybody figured out how to force the Alexa Java Client to use the Bluetooth speaker? Thanks in advance.

**Comment by dfoulkes** [comment]
Which version of Raspberian are you using? I had this issue with another project.

Have you tried to change the default setup on rasp-config?

Take a look at this: https://www.raspberrypi.org/documentation/configuration/audio-config.md

**Comment by s10dude** [comment]
Thank you for your reply. I downloaded the latest NOOBS and then ran sudo apt-get install update and upgrade before starting. I did try the raspi-config, but only saw three options. Auto, HDMI and 3.5mm. The built in Bluetooth is not listed as an option. I also tried forcing it to the 3.5mm audio, but that did not work either. It seems Alexa only wants to use HDMI no matter what I do. :(

**Comment by kbarto323** [comment]
I was able to get echo output via Bluetooth speaker following this tutorial - -http://plugable.com/2016/03/14/listening-to-bluetooth-audio-on-your-raspberry-pi-3-pi-2-or-pi-zero

**Comment by ajotwani** [comment]
We're closing this Issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own.

Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the Issue.

Thanks!

**Comment by junhocho** [comment]
I am really having trouble with my bluetooth speaker. VLC, youtube sound is coming fomr BT speaker but alexa keeps coming out from 3.5mm jack.

Installed Pulseaudio, mute alsa, did nearly everything, cannot solve it.

**Comment by samuelgh** [comment]
I have the same issue. Everything else (in the same terminal I run the alexa  java sample app) routes the sound to the bluetooth speaker. But not alexa. I guess it's forced somewhere since you configure it during the installation.

**Comment by D3rpi** [comment]
Can we please reopen this?
I trying to achieve the same thing here. I use a bluetooth speaker in my set up which works fine for tunein, iheart and flash briefings but the beeps(from the java app) and some Alexa repsones still go to my 3.5mm jack. How do I route all the audio to bluetooth?

**Comment by JSONMartin** [comment]
I'd like to do this too, so please reopen :)

**Comment by D3rpi** [comment]
@ajot Can this be re-opened?
Thanks

**Comment by sunildm4u** [comment]
Please reopen this issue. This is a major step in using raspberrypi as echo

**Comment by z10nn** [comment]
I agree, please re-open this issue. We need to solve this.

**Comment by D3rpi** [comment]
Since this one is still closed, I opened a new issue about this topic. 
https://github.com/alexa/alexa-avs-sample-app/issues/730
